packages/services/emails/src/environment.ts

```diff
@@ -71,6 +71,20 @@ const PrometheusModel = zod.object({
   PROMETHEUS_METRICS_LABEL_INSTANCE: emptyString(zod.string().optional()),
 });
 
+const LogModel = zod.object({
+  LOG_LEVEL: zod
+    .union([
+      zod.literal('trace'),
+      zod.literal('debug'),
+      zod.literal('info'),
+      zod.literal('warn'),
+      zod.literal('error'),
+      zod.literal('fatal'),
+      zod.literal('silent'),
+    ])
+    .optional(),
```

Comment:
these should be wrapped in `emptyString()` so that `""` is also interpreted as an empty value rather than raise a runtime error.